doc/ranger.pod

```diff
@@ -1840,6 +1841,19 @@ I<highlight> will pick up command line options specified in this variable. A
 C<--style=> option specified here will override C<HIGHLIGHT_STYLE>. Similarly,
 C<--replace-tabs=> will override C<HIGHLIGHT_TABWIDTH>.
 
+=item OPENSCAD_COLORSCHEME
+
+Specifies the colorscheme used by I<openscad> while previewing 3d models. The
```

Comment:
I think 3D should be capitalized everywhere.

Reply:
`%s/3d/3D` done